ktor-features/ktor-gson/jvm/src/io/ktor/gson/GsonSupport.kt

```diff
@@ -6,6 +6,7 @@ package io.ktor.gson
 
 import com.google.gson.*
 import io.ktor.application.*
+import io.ktor.application.newapi.*
```

Comment:
Please consider avoiding ad-hoc code even in prototypes. The same applies for TODOs stuff as well